### DISCUSSION TITLE
How to set a custom background color?

### DISCUSSION THREAD

**Discussion by ARR4NN** [squidfunk/mkdocs-material#2951] [Q&A] [answered]
can someone give me a run down on adding my own background color, I guess I need to use custom css but I don't know how to. Could you please explain

**Comment by squidfunk** [comment, marked as answer]
Just follow the guide on [adding additional CSS](https://squidfunk.github.io/mkdocs-material/customization/#additional-css) and start with the following:

``` css
body {
  background: red;
  /* or, using CSS variables */
  --md-default-bg-color: red;
}
```

For all other questions regarding CSS, I recommend posting them on StackOverflow, as it's not specific to this project.

**Reply by ARR4NN** [reply]
Thankyou i will try it now

**Reply by ARR4NN** [reply]
Do you have a discord i can message you on

**Reply by ARR4NN** [reply]
i added it
![image](https://user-images.githubusercontent.com/70060472/130351274-ae650c78-1d4b-430c-8e11-53b883e4e21e.png)
![image](https://user-images.githubusercontent.com/70060472/130351284-eb829bf5-2194-40cc-8444-bffa3045be82.png)
![image](https://user-images.githubusercontent.com/70060472/130351288-e88daf43-a5ae-4361-84af-b4bd77953868.png)

**Reply by ARR4NN** [reply]
but it doesnt work

**Reply by ARR4NN** [reply]
```
[I 210822 11:11:14 handlers:82] Ignore: C:\Users\Ajhar\OneDrive\Documents\PEMC MAIN MASTER\wiki\docs\stylesheets\more.css
INFO    -  Ignore: C:\Users\Ajhar\OneDrive\Documents\PEMC MAIN MASTER\wiki\docs\stylesheets\more.css
```

**Reply by ARR4NN** [reply]
why does it ignore it

**Comment by ARR4NN** [comment]
any ideas?

**Reply by ARR4NN** [reply]
@squidfunk

**Comment by ARR4NN** [comment]
Nevermind i got it, sorry for spam

**Reply by squidfunk** [reply]
Great. I'm marking my original answer again.

**Reply by MaximilianKohler** [reply]
I did that but it overrides the dark theme option too. `   --md-default-bg-color: red;` seemed to have no impact. 

I found https://github.com/squidfunk/mkdocs-material/issues/2987 and looked at the "color definitions guide" link but didn't see anything that elucidates. 

I'm looking in the other files https://github.com/squidfunk/mkdocs-material/tree/master/src/assets/stylesheets

`// Define primary colors` here https://github.com/squidfunk/mkdocs-material/blob/master/src/assets/stylesheets/palette/_primary.scss looks too limited. IE: I want to make the default background light grey `#F0F1F3` instead of white. 

I'm guessing it's one of these (?), but I can't decipher that:
```
    --md-primary-fg-color--light:      hsl(#{hex2hsl(list.nth($colors, 2))});
      --md-primary-bg-color--light:    hsla(0, 0%, 0%, 0.54);
```

**Reply by squidfunk** [reply]
You can scope CSS overrides to the [light and dark schemes](https://squidfunk.github.io/mkdocs-material/setup/changing-the-colors/#color-scheme) – the selector applies to the body, so just set it there:

``` css
/* Light mode */
[data-md-color-scheme="default"] {
  --md-default-bg-color: red;
}

/* Dark mode */
[data-md-color-scheme="slate"] {
  --md-default-bg-color: blue;
}
```
